app/controllers/admin/base_controller.rb

```diff
@@ -20,16 +20,28 @@ def should_count?
         controller_name === "services" || "requests"
     end
 
-    def set_counts
-        @all_count = Service.kept.count
-        @ofsted_count = Service.kept.ofsted_registered.count
-        @pending_count = Service.kept.where(approved: nil).count
-        @archived_count = Service.discarded.count
-        if APP_CONFIG["directories"].present?
-            @directories_counts = {}
+    def set_counts        
+          @service_counts_all = {
+            all: Service.kept.count,
+            ofsted: Service.kept.ofsted_registered.count,
+            pending: Service.kept.where(approved: nil).count,
+            archived: Service.discarded.count
+          }
+          @service_counts = {}
+          @service_counts[:all] = @service_counts_all
```

Comment:
This might be a bit clearer, and avoids setting an instance variable that is only used once:

```suggestion
          @service_counts = {
            all: {
              all: Service.kept.count,
              ofsted: Service.kept.ofsted_registered.count,
              pending: Service.kept.where(approved: nil).count,
              archived: Service.discarded.count
            }
          }
```